.github/workflows/publish-abac-nuget.yml

```diff
@@ -0,0 +1,36 @@
+name: Pack and publish Altinn.Authorization.ABAC nuget package
+
+on:
+  pull_request:
+    branches: infrastructure/publish-nuget-action
+  release:
+    types:
+      - published
+
+jobs:
+  build-pack:
+    #if: startsWith(github.ref, 'refs/tags/Altinn.Authorization.ABAC-')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: |
+            2.0.x
+      - name: Install deps
```

Comment:
I find that github actions files are much easier to read if you put a newline between the steps, so I recommend doing that